config/500-webhook.yaml

```diff
@@ -38,9 +38,6 @@ spec:
         # This is the Go import path for the binary that is containerized
         # and substituted here.
         image: knative.dev/eventing/cmd/webhook
-        volumeMounts:
```

Comment:
Just noticed this removed the volumeMount without removing the volume definition from the podspec.

Reply:
@grantr thanks, I will fix it up in the change I'm working on now that is moving eventing over to the refactored reconciler-based version.  Good catch!